.github/workflows/pkr-bld-qemu-arm64.yml

```diff
@@ -35,21 +38,50 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@main
-#      - name: Setup Packer
-#        if: steps.verify-changed-files.outputs.files_changed == 'true'
-#        uses: hashicorp/setup-packer@main
-#        with:
-#          version: latest
+#       - name: Install qemu libvirt vagrant`
+#         run: |
+#          brew tap hashicorp/tap
+#          brew update
+#          brew install --cask hashicorp/tap/hashicorp-vagrant
+#          brew install qemu libvirt
+#          vagrant plugin install vagrant-libvirt vagrant-qemu
+#          brew services start libvirt
+ #     - name: Setup Packer
+ #       if: steps.verify-changed-files.outputs.files_changed == 'true'
+ #       uses: hashicorp/setup-packer@main
+ #       with:
+ #         version: latest
+#       - name: Install Chef
+#         uses: actionshub/chef-install@3.0.0
+#       - name: Setup Packer
+#         uses: hashicorp/setup-packer@main
+#         with:
+#           version: latest
+#       - name: Install Bento
+#         run: |
+#           eval "$(chef shell-init bash)"
+#           gem build bento.gemspec
+#           gem install bento-*.gem
       - name: Bento build
         run: |
           eval "$(chef shell-init bash)"
-          bento build -o ${{ matrix.provider }}.vm --vars 'ssh_timeout=30m' --vars "qemu_accelerator=hvf" os_pkrvars/$(echo ${{ matrix.os }} | cut -d "-" -f 1)/"${{ matrix.os }}-aarch64.pkrvars.hcl"
```

Comment:
This seems one of the key change that need to be perform in order to resolve this issue : https://github.com/chef/bento/issues/1627